.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish release
+on:
+  # Runs whenever a new tag is pushed
+  push:
+    tags:
+      - '*.*.*'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # Publish job
+  publish:
+    runs-on: ubuntu-latest
+    # Sets permissions of the GITHUB_TOKEN to allow publishing to GitHub Packages
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
+      - name: Validate Gradle wrapper
+        uses: gradle/actions/wrapper-validation@v3
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+      - name: Publish package
+        run: ./gradlew publish
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPOSILITE_USERNAME: ${{ secrets.REPOSILITE_USERNAME }}
+          REPOSILITE_PASSWORD: ${{ secrets.REPOSILITE_PASSWORD }}
```

Comment:
@georgechoustoulakis Can you help me set up this secret across the entire THEOplayer organization? Currently these credentials are only in [android-ui](https://github.com/THEOplayer/android-ui).

Reply:
Thanks to @erikdeweerdt for setting this up. 🙏